editor/project_converter_3_to_4.cpp

```diff
@@ -304,7 +304,7 @@ bool ProjectConverter3To4::convert() {
 	maximum_line_length = 10000; // Use only for tests bigger value, to not break them.
 
 	ERR_FAIL_COND_V_MSG(!test_array_names(), false, "Cannot start converting due to problems with data in arrays.");
-	ERR_FAIL_COND_V_MSG(!test_conversion(reg_container), false, "Cannot start converting due to problems with converting arrays.");
+	ERR_FAIL_COND_V_MSG(!test_conversion(reg_container), false, "Cannot start converting due to problems with converting.");
```

Comment:
That error message isn't very clear :D

Reply:
True :laughing:. I wasn't sure what to call it. It wasn't specifically "arrays", more just "converting stuff in general".

Reply:
"Can't start converting due to the errors above", maybe, since there should be at least one preceding error that explains the issue?

Reply:
Maybe "Aborting conversion due to validation tests failing."?

Reply:
Updated the message.